components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -5321,95 +5324,216 @@ def generate_imports(config, cgthings, descriptors, callbacks=None, dictionaries
-        'dom::bindings::codegen::{PrototypeList, RegisterBindings, UnionTypes}',
-        'dom::bindings::error::{Fallible, Error, ErrorResult}',
+        'dom::bindings::conversions::ConversionBehavior',
+        'dom::bindings::conversions::ConversionResult',
+        'dom::bindings::conversions::DOM_OBJECT_SLOT',
+        'dom::bindings::conversions::IDLInterface',
+        'dom::bindings::conversions::is_array_like',
+        'dom::bindings::conversions::FromJSValConvertible',
+        'dom::bindings::conversions::StringificationBehavior',
+        'dom::bindings::conversions::ToJSValConvertible',
+        'dom::bindings::conversions::jsid_to_str',
+        'dom::bindings::conversions::native_from_handlevalue',
+        'dom::bindings::conversions::native_from_object',
+        'dom::bindings::conversions::private_from_object',
+        'dom::bindings::conversions::root_from_object',
+        'dom::bindings::conversions::root_from_handleobject',
+        'dom::bindings::conversions::root_from_handlevalue',
+        'dom::bindings::codegen::PrototypeList',
+        'dom::bindings::codegen::RegisterBindings',
+        'dom::bindings::codegen::UnionTypes',
+        'dom::bindings::error::Fallible',
+        'dom::bindings::error::Error',
+        'dom::bindings::error::ErrorResult',
         'dom::bindings::error::Error::JSFailed',
         'dom::bindings::error::throw_dom_exception',
-        'dom::bindings::guard::{Condition, Guard}',
+        'dom::bindings::guard::Condition',
+        'dom::bindings::guard::Guard',
         'dom::bindings::proxyhandler',
-        'dom::bindings::proxyhandler::{ensure_expando_object, fill_property_descriptor}',
-        'dom::bindings::proxyhandler::{get_expando_object, get_property_descriptor}',
+        'dom::bindings::proxyhandler::ensure_expando_object',
+        'dom::bindings::proxyhandler::fill_property_descriptor',
+        'dom::bindings::proxyhandler::get_expando_object',
+        'dom::bindings::proxyhandler::get_property_descriptor',
         'dom::bindings::num::Finite',
-        'dom::bindings::str::{ByteString, DOMString, USVString}',
-        'dom::bindings::weakref::{DOM_WEAK_SLOT, WeakBox, WeakReferenceable}',
+        'dom::bindings::str::ByteString',
+        'dom::bindings::str::DOMString',
+        'dom::bindings::str::USVString',
+        'dom::bindings::weakref::DOM_WEAK_SLOT',
+        'dom::bindings::weakref::WeakBox',
+        'dom::bindings::weakref::WeakReferenceable',
         'dom::browsingcontext::BrowsingContext',
         'mem::heap_size_of_raw_self_and_children',
         'libc',
         'util::prefs::PREFS',
-        'script_runtime::{store_panic_result, maybe_take_panic_result}',
+        'script_runtime::store_panic_result',
+        'script_runtime::maybe_take_panic_result',
         'std::borrow::ToOwned',
         'std::cmp',
         'std::mem',
         'std::num',
         'std::os',
-        'std::panic::{self, AssertUnwindSafe}',
+        'std::panic::{self}',
```

Comment:
Ah, right, you need to remove `::{self}` entirely, i.e. the final declaration would look like `std::panic`.